src/Integration.Vsix/InternalsVisibleTo.cs

```diff
@@ -21,9 +21,11 @@
 using System.Runtime.CompilerServices;
 
 #if SignAssembly
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")] // Moq
```

Comment:
I don't think this will work: if the main assembly is signed then all of the assemblies it references have to be signed too. You'll need to add the public key: see http://www.blackwasp.co.uk/moqinternals.aspx